src/domain/LevelState.ts

```diff
@@ -0,0 +1,12 @@
+import { reactive } from 'vue'
+import type { Level } from './Level'
+
+export interface LevelState {
```

Comment:
```suggestion
export interface Level {
```

This is a domain class. A domain should not be aware of other parts of application (like state, or services)